cirq-core/cirq/protocols/decompose_protocol.py

```diff
@@ -93,16 +93,12 @@ class SupportsDecompose(Protocol):
     """An object that can be decomposed into simpler operations.
 
     All decomposition methods should ultimately terminate on basic 1-qubit and
-    2-qubit gates included by default in Cirq. Cirq does not make any guarantees
-    about what the final gate set is. Currently, decompositions within Cirq
-    happen to converge towards the X, Y, Z, CZ, PhasedX, specified-matrix gates,
-    and others. This set will vary from release to release. Because of this
-    variability, it is important for consumers of decomposition to look for
-    generic properties of gates, such as "two qubit gate with a unitary matrix",
-    instead of specific gate types such as CZ gates (though a consumer is
-    of course free to handle CZ gates in a special way, and consumers can
-    give an `intercepting_decomposer` to `cirq.decompose` that attempts to
-    target a specific gate set).
+    2-qubit gates included by default in Cirq. If a custom decomposition is not
+    specified, Cirq will decompose all operations to X/Y/Z/CZ/Measurement + Global
```

Comment:
I'm confused since these aren't universal?  Is it missing PhasedX?

Reply:
I meant `XPow/YPow/ZPow/CZPow`; updated the docstring to clarify this.